package.json

```diff
@@ -14,8 +14,10 @@
     "test": "mocha"
   },
   "files": [
-    "index.js"
+    "index.js",
+    "types/index.d.ts"
```

Comment:
Sorry to pester, can you please move it at the root? It's a single file, so I prefer to add a subfolder.

Reply:
It is usually convention to add it in a types folder that is where TypeScript developers will expect to find it. I can move it if you really want though

Reply:
Ok, no strong opinion.